### ISSUE TITLE
ftrace crashes when using abi.encode

### ISSUE THREAD

**Issue by quezak** [ConsenSys/surya#80]
If the contract uses `abi.encode`, `abi.encodePacked` (and probably any `abi` method), `surya ftrace` crashes:
```
...>: npx surya ftrace Test::test1 all Test.sol 
Cannot read property 'encodePacked' of undefined
...>: npx surya ftrace Test::test2 all Test.sol 
Cannot read property 'encode' of undefined
```

A simple reproduction contract:
```
pragma solidity ^0.5.3;

contract Test {
    function test1() external returns (bytes) {
        return abi.encodePacked(uint16(1), bytes32('test'));
    }
    function test2() external returns (bytes) {
        return abi.encode(uint16(1), bytes32('test'));
    }
}
```

**Comment by GNSPS** [comment]
Hey @quezak! Thank you so much for reporting! 🙌 

I was OOO for last week but will get on this ASAP!

**Comment by GNSPS** [comment]
It should be resolved with #83 😄 🎉 

Can you please confirm it is working? You should be able to install this dev version with:
`npm install -g surya@next`

**Comment by quezak** [comment]
Yeah, it works now, thanks!
